src/components/project/list.vue

```diff
@@ -80,12 +133,23 @@ export default {
 </script>
 
 <style lang="scss">
-#project-list table { table-layout: fixed; }
+@import '../../assets/scss/mixins';
+
+#archived-projects {
+  .title {
+    a { @include text-link; }
+    color: $color-action-foreground;
+    font-size: 24px;
+    font-weight: 500;
+  }
+}
+
 </style>
 
 <i18n lang="json5">
 {
   "en": {
+    "archived": "Archived Projects",
```

Comment:
I usually add a comment when a message is used as a title, in case the translator needs to use title case. For example, "This is a title shown above a section of the page."